coprocessor/scripts/deploy-smart-contracts.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Exit the script immediately if any command fails
+set -e
```

Comment:
this might be problematic for etherscan verification we noticed with Marcus yesterday

Reply:
we are in local e2e it will help us to know if there is any failure.